modules/terrain/docs/README.md

```diff
@@ -11,7 +16,14 @@ npm install @loaders.gl/core
 
 ## Attribution
 
-`@loaders.gl/terrain` uses [MARTINI](https://github.com/mapbox/martini) for mesh reconstruction.
+The `TerrainLoader` uses [MARTINI](https://github.com/mapbox/martini) for mesh
+reconstruction.
+
+The `QuantizedMeshLoader` uses
+[`quantized-mesh-decoder`][quantized-mesh-decoder] from HERE under the MIT
+license to decode quantized mesh.
+
+[quantized-mesh-decoder]: https://github.com/heremaps/quantized-mesh-decoder
```

Comment:
Oh it's a habit of mine when writing Markdown to reflow lines to keep under 80 characters per line, since the break doesn't mean anything in standard (Commonmark) Markdown (though it does in Github Markdown, at least in issues).

Anyways I can take out the breaks

Reply:
It was less about that and more about the `ISC license` at the end being stranded.

Reply:
Oh I think that was meant to be attached to the Martini line, since Martini is licensed ISC and `quantized-mesh-decoder` is MIT.